### PR TITLE
mobile: make deposit page copy more explicit

### PR DESCRIPTION
**Pull request by nalinbhardwaj**

### CODE REVIEW

File: packages/daimo-api/src/api/getAccountHistory.ts [hunk 1]
```diff
@@ -200,10 +200,19 @@ function fetchRecommendedExchanges(account: EAccount): RecommendedExchange[] {
...
-    { cta: "Deposit from Coinbase", url: cbUrl },
-    { cta: "Deposit from Ramp", url: getRampNetworkURL(account) },
+    {
+      title: "Transfer from Coinbase & other options",
+      cta: "Deposit from Coinbase",
+      url: cbUrl,
+    },
+    {
+      title: "Cards, banks, & international options",
+      cta: "Buy USDC from Ramp",
```
Comment (dcposch) [inline packages/daimo-api/src/api/getAccountHistory.ts:214]: BUY USDC

File: packages/daimo-api/src/api/getAccountHistory.ts [hunk 2]
```diff
@@ -200,10 +200,19 @@ function fetchRecommendedExchanges(account: EAccount): RecommendedExchange[] {
 
   return [
     {
+      title: "Transfer another token to Daimo",
```
Comment (dcposch) [inline packages/daimo-api/src/api/getAccountHistory.ts:203]: Transfer from Ethereum

**Review by dcposch — COMMENTED** [review]
🚀

File: packages/daimo-api/src/api/getAccountHistory.ts [hunk 3]
```diff
@@ -200,10 +200,19 @@ function fetchRecommendedExchanges(account: EAccount): RecommendedExchange[] {
...
       cta: "Bridge from any wallet",
       url: `https://daimo.com/bridge/${account.name}`,
     },
-    { cta: "Deposit from Coinbase", url: cbUrl },
-    { cta: "Deposit from Ramp", url: getRampNetworkURL(account) },
+    {
+      title: "Transfer from Coinbase & other options",
```
Comment (dcposch) [inline packages/daimo-api/src/api/getAccountHistory.ts:208]: Send from Coinbase & other options

**Review by dcposch — APPROVED** [review]

File: packages/daimo-api/src/api/getAccountHistory.ts (hunk 1 above)
Comment (nalinbhardwaj) [reply packages/daimo-api/src/api/getAccountHistory.ts]: done

File: packages/daimo-api/src/api/getAccountHistory.ts (hunk 2 above)
Comment (nalinbhardwaj) [reply packages/daimo-api/src/api/getAccountHistory.ts]: done

**Pull request merged**